src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -100,15 +107,15 @@ protected override OperationStatus<IMethodSymbol> GenerateMethodDefinition(bool
                     name: _methodName.ToString(),
                     typeParameters: CreateMethodTypeParameters(),
                     parameters: CreateMethodParameters(),
-                    statements: result.Data,
-                    methodKind: localFunction ? MethodKind.LocalFunction : MethodKind.Ordinary);
+                    statements: statements.CastArray<SyntaxNode>(),
+                    methodKind: this.LocalFunction ? MethodKind.LocalFunction : MethodKind.Ordinary);
 
-                return result.With(
-                    MethodDefinitionAnnotation.AddAnnotationToSymbol(
-                        Formatter.Annotation.AddAnnotationToSymbol(methodSymbol)));
+                return MethodDefinitionAnnotation.AddAnnotationToSymbol(
+                    Formatter.Annotation.AddAnnotationToSymbol(methodSymbol));
             }
 
             protected override async Task<SyntaxNode> GenerateBodyForCallSiteContainerAsync(
+                SyntaxNode context,
```

Comment:
this is the same as `SyntaxNode insertionPointNode`.  but i acll it `context` in some places because that's the original name in the code.  i'll fix up to make this all consistent in followup.

Reply:
renamed in mop up.